backend/api/build.gradle.kts

```diff
@@ -1,20 +1,10 @@
-//val lombok: Unit by extra
-dependencies {
-            compileOnly(rootProject.libs.lombok)
-            annotationProcessor(rootProject.libs.lombok)
-            testCompileOnly(rootProject.libs.lombok)
-            testAnnotationProcessor(rootProject.libs.lombok)
-        }
-
-//val vavr: Unit by extra
-dependencies {
-                // Unlike any other current dependency, Vavr classes are very likely to end up in binary interface of the depending subproject,
-                // hence it's better to just treat Vavr as an `api` and not `implementation` dependency by default.
-                api(rootProject.libs.vavr)
-            }
-
+import com.virtuslab.gitmachete.buildsrc.BuildUtils
 
 dependencies {
     api(project(":qual"))
     api(project(":branchLayout:api"))
 }
+
+BuildUtils.lombok(project)
```

Comment:
<3

Reply:
Nit: maybe just `import com.virtuslab.gitmachete.buildsrc.BuildUtils.*` and `lombok(project)`?

Also, is there a way to call just `lombok()`? Maybe adding an extension method to `project`'s class? I assume that e.g. the above `dependencies` are probably called in a similar way (?)

Reply:
That's a good idea, I'm moving that also to #1004